AsyncDisplayKit/ASImageNode+AnimatedImage.h

```diff
@@ -0,0 +1,15 @@
+//
+//  ASImageNode+AnimatedImage.h
+//  AsyncDisplayKit
+//
+//  Created by Garrett Moon on 3/22/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import "ASImageNode.h"
+#import "ASImageProtocols.h"
+
+@interface ASImageNode ()
```

Comment:
Does this need to be in a separate file?  To make it more discoverable, we might want to put it in the main header - at most perhaps a separately named category ?